README.md

```diff
@@ -4,6 +4,12 @@
 
 ![System Settings Screenshot](data/screenshot.png?raw=true)
 
+## Plugs
+
+Switchboard is just the container application for Switchboard Plugs, which provide the actual settings for various hardware and software.
+
+[Browse all Plugs](https://github.com/elementary?q=switchboard-plug)
```

Comment:
I was confused for a moment because it simply seemed to bring me to the regular elementary organization. You could add the following anchor to scroll directly to the results. 
https://github.com/elementary?q=switchboard-plug#org-repositories